src/main/java/mate/academy/spring/model/dto/request/ShoppingCartRequestDto.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.model.dto.request;
+
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+
+public class ShoppingCartRequestDto {
+    private List<Ticket> tickets;
```

Comment:
```suggestion
    private List<Long> ticketIds;
```
не треба приймати/передавати моделі в DTO-шках